protocol/flatbuffers/rpc.fbs

```diff
@@ -181,3 +188,31 @@ table AutoBoneEpochResponse {
     epoch_error: float32;
     adjusted_skeleton_parts: [SkeletonPart];
 }
+
+enum FkToggle: uint8 {
+    NONE = 0,
+    EXTENDED_SPINE_MODEL = 1,
+    EXTENDED_PELVIS_MODEL = 2,
+    EXTENDED_KNEE_MODEL = 3,
+    FORCE_ARMS_FROM_HMD = 4
+}
+
+table FkSkeletonToggle{
+    skeletonToggle: FkToggle;
+    value: bool;
+}
+
+enum FkValue: uint8{
+    NONE = 0,
+    WAIST_SPINE_AVERAGE = 1,
+    WAIST_PELVIS_AVERAGE = 2,
+    HIP_PELVIS_AVERAGE = 3,
+    PELVIS_HIP_AVERAGE = 4,
+    WAIST_TRACKER_PELVIS_AVERAGE = 5,
+    KNEE_TRACKER_ANKLE_AVERAGE = 6
+}
+
+table FkSkeletonValue{
```

Comment:
Does this mean that the "setting" in `FkValue` is set to the "value" in `value`? in that case why is the name of the setting - which represents a slot that a value is put into - called `FkValue`. Values go inside fields/properties/settings so this naming is confusing

Reply:
Same as toggle. They were named after what they are, but if you have a different naming in mind that's better we can change it.